DotNet/overrides.js

```diff
@@ -53,4 +53,7 @@ module.exports = {
     'lazy load page with all options specified with vg': {skipEmit: true},
     'lazy load page with default options': {skipEmit: true},
     'lazy load page with default options with vg': {skipEmit: true},
+
+    'should use regions padding': { skipEmit: true },
+    'should use regions padding with vg': { skipEmit: true },
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9618817)
```suggestion
    "should use regions padding with vg": { skipEmit: true },
```